repos/rust-lang/edition-guide.toml

```diff
@@ -12,6 +12,6 @@ pattern = "master"
 ci-checks = ["Success gate"]
 required-approvals = 0
 
-[[environments]]
-name = "github-pages"
+[environments.github-pages]
+branches = []
```

Comment:
Just to avoid a common default case, if the branches array is empty, I'd remove it from the current files, and keep the default.